portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,48 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomQuote() {
+  const quotes =
+      ['Let your smile change the world, dont let the world change your smile :)',
+       'The best is yet to be!',
+       'You dont have to be great to start, but you have to start to be great!',
+       'When you cannot find the sunshine, be the SUNSHINE!',
+       'Dont stop until you are proud'];
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const quoteContainer = document.getElementById('quote-container');
+  quoteContainer.innerText = quote;
 }
+window.onload = function () {
+    startTab(); }
+
+function startTab() {
+    document.getElementById("defaultOpen").click();}
+    
+function openPage(pageName, elmnt, color) {
+  
+  // Hide all elements with class="tabcontent" by default */
+  var i, tabcontent, tablinks;
```

Comment:
Nit: unlike C, in JS, we usually declare and initiate variables at the place where they are needed.